mdtraj/formats/prmtop.py

```diff
@@ -160,8 +163,11 @@ def load_prmtop(filename, **kwargs):
                         raw_data[flag].append(item.strip())
 
     # Add atoms to the topology
-
     pdb.PDBTrajectoryFile._loadNameReplacementTables()
+    if not standard_names:
+        pdb.PDBTrajectoryFile._residueNameReplacements = {}
```

Comment:
My apologies again! 

Upon closer review on this, I have to flag this. `PDBTrajectoryFile._residueNameReplacemets` and `PDBTrajectoryFile._atomNameReplacements` are both class attributes in `PDBTrajectoryFile`. This essentially means that changing them would globally change the name replacement tables for this class *and all instances of the class*, which may produce some very unintended side effects. These attributes should not be modified here.